packages/remark-captions/__tests__/index.js

```diff
@@ -6,6 +6,7 @@ import remark2rehype from 'remark-rehype'
 
 import remarkCaptions from '../src/'
 import remarkGridTables from '../../remark-grid-tables/src'
+const remarkStringify = require('remark-stringify')
```

Comment:
This should be an `import` placed above, see

https://github.com/artragis/zmarkdown/blob/b417f74a08e959cdb1a69e4075a20abab5ae66b3/packages/remark-captions/__tests__/index.js#L1-L6